### ISSUE TITLE
replace call to functions from the string module in lore

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#4994]
|<img alt="nueces's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">| nueces reported|
|-|-|
|Trac ID|trac#4994|
|Type|enhancement|
|Created|2011-03-27 17:10:28Z|
|Branch|https://github.com/twisted/twisted/tree/str-methods-lore-4994|

To make the port to python 3k more easy, I'm replacing all the calls to functions from the string module, for their equivalent from the str type object or calling directly to the built-in method of the string variable.

Attachments:

* [lore.patch](https://raw.githubusercontent.com/twisted/twistedmatrix.com-trac-attachments/trunk/ticket/7d8/7d803a56c0e22b6c81011b87e1ce43dd8704ead6/880463956a8076f5e261c3863e95d8786d74ac17.patch) (1735 bytes) - added by nueces on 2011-03-27 17:10:39Z - patch

<details><summary>Searchable metadata</summary>

```
trac-id__4994 4994
type__enhancement enhancement
reporter__nueces nueces
priority__normal normal
milestone__ 
branch__branches_str_methods_lore_4994 branches/str-methods-lore-4994
branch_author__lvh lvh
status__closed closed
resolution__fixed fixed
component__lore lore
keywords__py3k py3k
time__1301245828000000 1301245828000000
changetime__1302056432000000 1302056432000000
version__None None
owner__lvh lvh

```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="lvh's avatar" src="https://avatars.githubusercontent.com/u/97816?s=50" width="50" height="50">](https://github.com/lvh)|@lvh set owner to @lvh|
|-|-|

**Comment by twisted-trac** [comment]
|[<img alt="lvh's avatar" src="https://avatars.githubusercontent.com/u/97816?s=50" width="50" height="50">](https://github.com/lvh)<a name="note_3"></a>|@lvh commented|
|-|-|

(In [31449]) Branching to 'str-methods-lore-4994'

**Comment by twisted-trac** [comment]
|[<img alt="lvh's avatar" src="https://avatars.githubusercontent.com/u/97816?s=50" width="50" height="50">](https://github.com/lvh)<a name="note_4"></a>|@lvh commented|
|-|-|

Buildbot looks clean! Needs a topfile, but since this is just a misc change, I'll do that. Adding to my merge queue.

Thank you for your contribution!

**Comment by twisted-trac** [comment]
|[<img alt="lvh's avatar" src="https://avatars.githubusercontent.com/u/97816?s=50" width="50" height="50">](https://github.com/lvh)<a name="note_5"></a>|@lvh set status to `closed`|
|-|-|

(In [31558]) Replaced calls to functions from the string module in twisted.lore.

Author: nueces
Reviewers: lvh
Fixes: [#4994](https://github.com/twisted/twisted/issues/4994)

twisted.lore was previously using deprecated calls to functions in the string
module, which are deprecated in favor of str methods.
